lib/protobuf/active_record/transformation.rb

```diff
@@ -93,20 +94,27 @@ def _protobuf_convert_fields_to_attributes(key, value)
         #     # Do some blocky stuff...
         #   end
         #
-        def attribute_from_proto(attribute, transformer = nil, &block)
-          transformer ||= block
+        #   attribute_from_proto :status, lambda { |proto| nil }, :nullify_on => :status
+        #   attribute_from_proto :status, :nullify_on => :status do |proto|
+        #     nil
+        #   end
+        #
+        def attribute_from_proto(attribute, *args, &block)
+          options = args.extract_options!
+          transformation = args.first || block
```

Comment:
Or we can clean it up even more with something like this:

```ruby
callable = args.first || block

if callable.is_a?(Symbol)
  callable = lambda { |value| self.__send__(callable, value) }
else
  raise AttributeTransformerError unless callable.respond_to?(:call)
end
```

It will skip the callable check when we know that we've just created something that is callable.